packages/jest-snapshot/src/matcher.js

```diff
@@ -1,94 +0,0 @@
-/**
- * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @flow
- */
-'use strict';
-
-import type {Path} from 'types/Config';
-import type {SnapshotState} from './SnapshotState';
-
-const diff = require('jest-diff');
-
-type CompareResult = {
-  pass: boolean,
-  message: ?string,
-};
-
-module.exports = (
-  filePath: Path,
-  options: Object,
-  snapshotState: SnapshotState,
-) => (util: any, customEquality: any) => {
-  return {
-    negativeCompare() {
-      throw new Error(
-        'Jest: `.not` can not be used with `.toMatchSnapshot()`.',
-      );
-    },
-    compare(actual: any, expected: any): CompareResult {
-      if (expected !== undefined) {
-        throw new Error(
```

Comment:
It looks like you forgot to add a call to `ensureNoExpected` to the updated matcher.